templates/community-viz/package.json

```diff
@@ -5,14 +5,15 @@
     "jsFile": "index.js",
     "jsonFile": "index.json",
     "cssFile": "index.css",
-    "manifestFile": "manifest.json"
+    "manifestFile": "manifest.json",
+    "print": "printMessage.js"
   },
   "scripts": {
-    "build:dev": "node scripts/build.js --prod false",
-    "build:prod": "node scripts/build.js --prod true",
-    "push:dev": "node scripts/push.js --prod false",
-    "push:prod": "node scripts/push.js --prod true",
-    "watch": "webpack --watch",
+    "build:dev": "node scripts/dscc-scripts.js build prod:false",
```

Comment:
We can fix this later, but it's better to use normal-looking command arguments. i.e. `node path build prod=false` etc.

Reply:
Noted for later